### ISSUE TITLE
`Pattern not found` is constantly printed in insert mode. If I comment out `set noshowmode` the problem goes away.

### ISSUE THREAD

**Issue by thomassantosh** [neovim/neovim#19919]
### Neovim version (nvim -v)

NVIM v0.7.2

### Vim (not Nvim) behaves the same?

No

### Operating system/version

macOS 12.5.1

### Terminal name/version

Terminal.app

### $TERM environment variable

xterm-256color

### Installation

Homebrew

### How to reproduce the issue

To reproduce the issue, I merely type and I keep getting `pattern not found` after a few clicks. Right before it shows up, on the command bar, with the `vim-airline` plugin it shows `INSERT COMPL GENERIC` and then highlights `Pattern not found`. This only started occurring in the past week. 

When I comment out `set noshowmode` this seems to go away. 

As mentioned, I have `vim-airline` installed. However, this issue persists whether the plugin loads or not.

### Expected behavior

Basically, anything that doesn't keep spiraling into showing `pattern not found`. This somehow does not happen when I comment out `set noshowmode`.

![expected](https://user-images.githubusercontent.com/12192561/186290255-b0c1eb9d-b3e6-45ec-ba76-7ef62b8c50af.gif)

### Actual behavior

1. This feels similar to this issue: https://github.com/hrsh7th/nvim-cmp/issues/653.
2. This started about a week ago.

![2022-08-23 17 37 38](https://user-images.githubusercontent.com/12192561/186289879-380fff24-22a5-4866-909b-ceb60b29e88b.gif)

**Comment by Shougo** [comment]
Related issue: https://github.com/Shougo/deoplete.nvim/pull/1199

I think `git bisect` is needed for it.

**Comment by zeertzjq** [comment]
This happens in v0.7.2, so it isn't introduce by a recent change in Nvim.

**Comment by zeertzjq** [comment]
Closing as this is apparently a plugin problem.

**Comment by thomassantosh** [comment]
@zeertzjq - I don't have the `nvim-cmp` plugin installed. These are the plugins in my init.vim file.
``
call plug#begin('~/.config/nvim/plugged')
Plug 'vimwiki/vimwiki'
Plug 'Shougo/deoplete.nvim', { 'do': ':UpdateRemotePlugins' }
Plug 'zchee/deoplete-jedi'
Plug 'morhetz/gruvbox'
Plug 'vim-airline/vim-airline'
Plug 'jiangmiao/auto-pairs'
Plug 'scrooloose/nerdtree'
Plug 'nvim-lua/popup.nvim'
Plug 'nvim-lua/plenary.nvim'
Plug 'nvim-telescope/telescope.nvim'
call plug#end()
``

**Comment by zeertzjq** [comment]
Still no minimal enough steps to reproduce.

**Comment by thomassantosh** [comment]
Should I provide more detail and if so, what else can I clarify? I was hoping the gif files would help illustrate the issues. There's a marked difference between the first gif where the `set noshowmode` is commented out, and when it is active.

**Comment by Shougo** [comment]
@thomassantosh OK.  So, the plugin problem is fixed in https://github.com/Shougo/deoplete.nvim/pull/1199.
Please update deoplete to the latest version.

**Comment by thomassantosh** [comment]
@Shougo - you're a rockstar. Thank you. It works fine now.
